bears/generate_package.py

```diff
@@ -94,24 +103,64 @@ def create_upload_parser():
                         action='store_true')
     parser.add_argument('-u', '--upload', help='Upload the packages on PyPi',
                         action='store_true')
+    parser.add_argument('-c', '--conda', help='Create a conda package',
+                        metavar='dir')
     return parser
 
 
+def get_bear_glob(path):
```

Comment:
This doesn't work the expected way. `path` will either contain a path or `""`.
```
>>> def a(path="Bears"):
...   print(path)
... 
>>> a()
Bears
>>> a("")

>>> 
```
I will still have to do an if else check